README.md

```diff
@@ -71,8 +85,11 @@ override func viewDidLoad() {
 }
 ```
 
+### User Location
+The Inspetor iOS Library can use the user location to help us provide more accurate results, but it will **never** ask for the user location. If your app has already the permission for the user location the library will automatically capture the user location, othewise it won't send the location to us.
```

Comment:
I'd write `If your app has already the permission` as `If your app already has the permission`

Reply:
Also, I think `user location` is a bit repetitive here. I'd write as:

```
The Inspetor iOS Library can use user location to help us provide more accurate results, but it will **never** ask the user for it. If your app already has the permission for accessing user location the library will automatically capture it, otherwise it won't send the location to us.
```

(there was also a typo in othewise)